src/pheval/analyse/generate_summary_outputs.py

```diff
@@ -134,9 +116,10 @@ def generate_benchmark_variant_output(
     prioritisation_data: TrackRunPrioritisation, plot_type: str
 ) -> None:
     """Generate variant prioritisation outputs for benchmarking single run."""
-    RankComparisonGenerator(
-        prioritisation_data.variant_prioritisation.ranks
-    ).generate_variant_output(f"{prioritisation_data.variant_prioritisation.results_dir.name}")
+    RankComparisonGenerator(prioritisation_data.variant_prioritisation.ranks).generate_output(
```

Comment:
@julesjacobsen I am sure this is one of the things where you believe Python has gone wrong over a beautiful Java OO design..

Reply:
Nope. The previous implementation was all useless repetition. No one wants that.

Reply:
I meant prioritisation_data.variant_prioritisation.ranks without encapsulation!